src/content/certifications/exam-faqs/vault-operations-professional.mdx

```diff
@@ -11,7 +11,13 @@ We strongly recommend passing the associate-level Vault exam before taking the p
 - Information security fundamentals such as network security and RBAC
 - Understand the concepts and functionality of infrastructure running in containers including starting and stopping services, and reading logs
 
-![](https://www.datocms-assets.com/2885/1671559871-vault-badge-image-export.svg)
+<ThemedImage
+	lightSrc="https://www.datocms-assets.com/2885/1680025449-vault_cert_lt-1.png"
+	darkSrc="https://www.datocms-assets.com/2885/1680025429-vault_cert_dt-1.png"
```

Comment:
This is the workaround, I originally had this interface to accept an object like:
```
src={{
    light: '////',
    dark: '////'
}}
```
Which I still think is a better interface but for the sake of just getting this done for the upcoming release I'm rolling with this workaround.